apps/system/locales/system.en-US.properties

```diff
@@ -201,7 +201,7 @@ wantToInstallNow=Do you want to install the update now? You will not be able to
 installNow=Install Now
 notNow=Not Now
 # the <span> element will be displayed when the download finishes
-downloadingUpdateMessage.innerHTML=Downloading updates… <span>{{ progress }} downloaded.</span>
+downloadingUpdateMessage=Downloading updates… <span>{{ progress }} downloaded.</span>
```

Comment:
Yes, but since we're switching to DOM overlays, it's an opportunity to clean the code which previously had to handle that `span` every time the language changes.  Or maybe there is no such code and it's a simple `parent.complete span { display: inline; }` or someting.  I don't know, hence the question :)